hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -239,16 +241,16 @@ spec = parallel $ do
 
   describe "ChainSyncHandler" $ do
     prop "yields observed transactions rolling forward" $ do
-      forAllSt $ \(SomeOnChainHeadState -> st) tx -> do
+      forAll genChainStateWithTx $ \(st, tx, _) -> do
         let callback = \case
               Rollback{} ->
                 fail "rolled back but expected roll forward."
               Observation OnCloseTx{snapshotNumber} ->
                 -- FIXME: Special case for `OnCloseTx` because we don't directly observe the remaining contestation period,
                 -- it's the result of a computation that involves current time
-                fst <$> observeSomeTx tx st `shouldBe` Just OnCloseTx{snapshotNumber, remainingContestationPeriod = 0}
+                fst <$> observeAllTx tx st `shouldBe` Just OnCloseTx{snapshotNumber, remainingContestationPeriod = 0}
```

Comment:
Question: Why not keep `observeSomeTx` name? It seems ok to me and would remove some changes

Reply:
Good point. It was just useful when they co-existed.